registry/storage/driver/gcs/gcs_test.go

```diff
@@ -76,6 +74,7 @@ func init() {
 			privateKey:    privateKey,
 			client:        oauth2.NewClient(dcontext.Background(), ts),
 			chunkSize:     defaultChunkSize,
+			maxConcurrency: defaultMaxConcurrency,
```

Comment:
This isn't a bad change, but this is out of scope for this PR. This should be handled in a separate PR so that we can better track it in git history.